### PR TITLE
Have a single source of change log (instead of .md vs .html)

### PR DESCRIPTION
**Pull request by soncaokim**
Present to the user the CHANGELOG.md

The main motivation is to avoid forgetting to update the .html version (it happened on v1.4.0 and v1.4.1 - mea culpa).

Use a Markdown rendering lib:
- ~https://github.com/GrenderG/MarkdownView~
  Implemented with https://github.com/VinylMusicPlayer/VinylMusicPlayer/pull/665/commits/6a68ad1e948012f6cba96ba288e3d1faa35efbe0
  Attn: This requires a bunch of JS libraries, and they are not all up-to-date (-> risk of vulnerabilities) 
- ~https://github.com/mukeshsolanki/MarkdownView-Android~
  Considered, but discarded as it requies minSdk = 21
- https://github.com/noties/Markwon
  Implemented with https://github.com/VinylMusicPlayer/VinylMusicPlayer/pull/665/commits/0c9fe94df8bb02993665371571f0d6c2cb132508
  Library quite powerful, good documentation.

Todos
- [x] The old changelog had better margin and without highlight below title seems cleaner
- [x] The double highlight on title seems like a bug

### CODE REVIEW

**Comment by AdrienPoupa** [comment]
Good idea!

**Review by AdrienPoupa — APPROVED** [review]

**Comment by soncaokim** [comment]
I've added some changes to render the .md in the recent commits, and left my note on the chosen library in the PR summary.

If there is no veto, I'll merge this PR.

File: app/src/main/assets/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1 @@
+../../../../CHANGELOG.md
```
Comment (AdrienPoupa) [inline app/src/main/assets/CHANGELOG.md:1]: is this a symlink?

**Comment by AdrienPoupa** [comment]
All good on my end, thank you. Are you using a symlink to load the changelog from the assets folder?

File: app/src/main/assets/CHANGELOG.md (hunk 1 above)
Comment (soncaokim) [reply app/src/main/assets/CHANGELOG.md:1]: indeed

Comment (soncaokim) [reply app/src/main/assets/CHANGELOG.md:1]: This will work on Linux and MacOS, but not on WIndows.
Would that be a problem?

**Comment by Octoton** [comment]
Do you have control on the graphics settings ?
The old changelog had better margin and without highlight below title seems cleaner to me.
Old:
Old

New:
New

**Comment by Octoton** [comment]
The double highlight on title seems like a bug

**Comment by soncaokim** [comment]
> Do you have control on the graphics settings ?
> The old changelog had better margin and without highlight below title seems cleaner to me.

This should be fixable (the lib is well documented and is quite customizable). But is this really that annoying to you? :-)

> The double highlight on title seems like a bug

Indeed, I might have an idea where it comes from (auto linkify text - it detects a phone number here)

**Comment by Octoton** [comment]
> This should be fixable (the lib is well documented and is quite customizable). But is this really that annoying to you? :-)

Well it is the first popup people will see after upgrading, for me there need to be a margin before the "changelog" title.
Other than that, it is just personal tastes to remove all highlight I can find so clearly do as you wish.

**Comment by soncaokim** [comment]
@Octoton I've tried to tackle the comments you've left. The MD heading single underline is still there though, since it is part of the built-in styling.

Please have a second look.

**Comment by Octoton** [comment]
It is definitely better
Don't want to be this guy but is it possible to just add to the top margin ? Not left, right or bottom has it just look like wasted space.
On the previous commit the left, right and bottom where perfect.

**Comment by soncaokim** [comment]
> It is definitely better Don't want to be this guy but is it possible to just add to the top margin ? Not left, right or bottom has it just look like wasted space. On the previous commit the left, right and bottom where perfect.

I've made the change - https://github.com/VinylMusicPlayer/VinylMusicPlayer/pull/665/commits/197c3e5ca7145de06c80362d01f0ec5268570c58

**Comment by Octoton** [comment]
Thanks for the change, it is great

**Comment by Octoton** [comment]
Just a small question, do we want to put the PR link of each change in the changelog ?
Because when comparing version 141 with 130 or before, one is lot more simple to read but will give a little bit less information (not sure that the link is a useful info for the end user).

**Comment by soncaokim** [comment]
For each version, the changelog is auto generated by github and I've just
reviewed then  copied/pasted here.

There are possibilities to tune that further, ie
- on how the data are collected (via .yml conf, or via github actions
workflows)
- on how the data are rendered, ie shortenting the link the way they are
shown in Github website (via the Markdown library).

Let's go with the current implementation. This helps us to cut one manual
step in maintaining a .html file.

**Pull request merged**